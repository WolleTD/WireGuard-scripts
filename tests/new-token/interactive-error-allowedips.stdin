peer1
172.16..1/32
peer1
172.16.1.1/32
n
