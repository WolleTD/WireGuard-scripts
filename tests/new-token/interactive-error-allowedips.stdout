Add wg-setup tokens to /tmp/tmp\..+/toklist
Name: AllowedIPs: Error: Invalid AllowedIPs \(<addr>\[/cidr\]\[,<addr>\[/cidr\]\[,...\]\]\)
Name: AllowedIPs: 
[A-Z2-7]{8}		172\.16\.1\.1/32		peer1
Add another token\? \[y/N\] 
